### DISCUSSION TITLE
Open Ulauncher from tray icon?

### DISCUSSION THREAD

**Discussion by seidler2547** [Ulauncher/Ulauncher#1067] [Ideas]
Hi, I would like to use Ulauncher on a tablet device and so I was lookinng for a way to open it with a single tap. I found that tapping/clicking the tray icon only opens the menu, can it be configured to show the menu on right click and open the input field on left click?

**Comment by friday** [comment]
I agree that would be an improvement, and I have seen others do this so I know it's possible. And the discussion is concrete enough, so I created an issue for it: https://github.com/Ulauncher/Ulauncher/issues/1068.

It's not a priority issue for me/us though, so if you want to see this added soon you'd have to submit a PR.

(Closing the discussion in favor of the issue)
